WordPress/Classes/Utility/AppLocalizedString.swift

```diff
@@ -0,0 +1,69 @@
+import SwiftUI
+
+
+extension Bundle {
+    /// Returns the Bundle for the host `.app`.
+    ///
+    /// - If this is called from code already located in the main app's bundle or from a Pod/Framework,
+    ///   this will return the same as `Bundle.main`, aka the bundle of the app itself.
+    /// - If this is called from an App Extension (Widget, ShareExtension, etc), this will return the bundle of the
+    ///   main app hostsing said App Extension (while `Bundle.main` would return the App Extension itself)
+    ///
+    /// This is particularly useful to reference a resource or string bundled inside the app from an App Extension / Widget.
+    ///
+    /// - Note:
+    ///   In the context of Unit Tests this will likely return the Test Harness / Test Host app since that is the app running said tests.
```

Comment:
Confirmed 👍 

![image](https://user-images.githubusercontent.com/1218433/145066829-09dc7f7e-4bdf-42e0-b095-424e1e7e0401.png)

Reply:
```suggestion
    ///   In the context of Unit Tests this will return the Test Harness (aka Test Host) app, since that is the app running said tests.
```